packages/web-components/src/defaultSettings.ts

```diff
@@ -3,7 +3,7 @@ import { SelectedCategoryPropertiesSettings, SelectedProductPropertiesSettings }
 export const defaultProductProperties: Partial<SelectedProductPropertiesSettings> = {
     displayName: true,
     pricing: true,
-    dataKeys: ['ImageUrl', 'Url'],
+    dataKeys: ['ImageUrl', 'Url', 'relewise*'],
```

Comment:
Why has this been added? Do we have any jobs that prefix datakeys with Relewise?